frontend/src/app/data-entry/sources/sources.component.html

```diff
@@ -0,0 +1,26 @@
+<lc-breadcrumb [breadcrumbs]="breadcrumbs" />
+
+<h1>Data entry</h1>
+
+<p>Select a source to begin</p>
+
+<div class="list-group">
+    <a *ngFor="let source of sources$ | async; trackBy: identify"
+       class="list-group-item list-group-item-action"
+       [routerLink]="['source', source.id]">
+        <div class="d-flex align-items-center">
+            <h5 class="mb-1">{{ source.name }}</h5>
+            <small class="text-body-secondary ms-2">
+                ({source.numOfEpisodes, plural, =0 {No episodes} =1 {1 episode} other {{{ source.numOfEpisodes}} episodes}})
+            </small>
+        </div>
+        <p class="mb-1">
+            <ng-container *ngIf="source.medievalAuthor || source.medievalTitle">
+                {{ source.medievalAuthor }}<ng-container *ngIf="source.medievalAuthor && source.medievalTitle">, </ng-container><em>{{ source.medievalTitle }}</em>.
+            </ng-container>
+            <ng-container *ngIf="source.editionAuthor || source.editionTitle">
+                (Ed. {{ source.editionAuthor }}<ng-container *ngIf="source.editionAuthor && source.editionTitle">, </ng-container><em>{{ source.editionTitle }}</em>.)
+            </ng-container>
+        </p>
+    </a>
+</div>
```

Comment:
Two semantics issues:

- The header level in line 12 is incorrect; this would be `<h2>`. Though in this case, I think it may be more appropriate to code this section as a list (with `<ul>`/`<li>`) than with section headers.
- It's generally recommended that an anchor (`<a>`) does not contain too much text; this makes navigation by links harder. Some accessibility checkers will also complain about putting a header inside the link (you normally do it the other way around). In this case, you could  just put the `<a>` tag around the source name in line 12, and use a [stretched link](https://getbootstrap.com/docs/5.3/helpers/stretched-link/) to increase the clickable area.

So I would suggest something like this (please double check the layout, though):

```suggestion
<ul class="list-group">
    <li *ngFor="let source of sources$ | async; trackBy: identify"
       class="list-group-item list-group-item-action">
        <div class="d-flex align-items-center">
            <a [routerLink]="['source', source.id]" class="mb-1 stretched-link">{{ source.name }}</a>
            <small class="text-body-secondary ms-2">
                ({source.numOfEpisodes, plural, =0 {No episodes} =1 {1 episode} other {{{ source.numOfEpisodes}} episodes}})
            </small>
        </div>
        <p class="mb-1">
            <ng-container *ngIf="source.medievalAuthor || source.medievalTitle">
                {{ source.medievalAuthor }}<ng-container *ngIf="source.medievalAuthor && source.medievalTitle">, </ng-container><em>{{ source.medievalTitle }}</em>.
            </ng-container>
            <ng-container *ngIf="source.editionAuthor || source.editionTitle">
                (Ed. {{ source.editionAuthor }}<ng-container *ngIf="source.editionAuthor && source.editionTitle">, </ng-container><em>{{ source.editionTitle }}</em>.)
            </ng-container>
        </p>
    </li>
</ul>
```

Reply:
I swear I will actually remember to use the correct header hierarchy one of these days :P Both the recommendation of using shorter texts in anchor tags and Bootstrap's stretched link were new to me. Good to know! 👍

Reply:
I checked the layout and it still looks good :)